### PR TITLE
Add infinite scrolling example

### PR DESCRIPTION
**Pull request by amanmahajan7**

### CODE REVIEW

File: stories/demos/InfiniteScrolling.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+const loadMoreRowsClassname = css`
+  width: 180px;
+  padding: 8px 16px;
+  position: absolute;
+  bottom: 8px;
+  right: 8px;
+  color: white;
+  line-height: 35px;
+  background: rgba(0, 0, 0, 0.6);
```
Comment (nstepien) [inline stories/demos/InfiniteScrolling.tsx:15]:
```suggestion
  background: rgb(0 0 0 / 0.6);
```

File: stories/demos/InfiniteScrolling.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    setTimeout(() => resolve(newRows), 1000);
+  });
+}
+
+export function InfiniteScrolling() {
+  const [rows, setRows] = useState(() => createRows(50));
+  const [isLoading, setIsLoading] = useState(false);
+
+  async function handleScroll(event: React.UIEvent<HTMLDivElement>) {
+    if (!isAtBottom(event)) return;
```
Comment (nstepien) [inline stories/demos/InfiniteScrolling.tsx:94]:
```suggestion
    if (isLoading || !isAtBottom(event)) return;
```

**Review by nstepien — DISMISSED** [review]
I wonder if it's actually a good example, since it wouldn't work well with column sorting enabled 🤔

File: stories/demos/AllFeatures.tsx [hunk 3]
```diff
@@ -155,55 +144,34 @@ const columns: readonly Column<Row>[] = [
...
-
-    for (let i = 0; i < newRowsCount; i++) {
-      newRows[i] = createFakeRowObjectData(i + length);
-    }
-
-    setTimeout(() => resolve(newRows), 1000);
-  });
-}
-
 export function AllFeatures() {
```
[the remark this answers is not in the document]
Comment (amanmahajan7) [reply stories/demos/AllFeatures.tsx:172]: Only copy/paste and drag fill I believe. We can revisit this example

**Review by nstepien — APPROVED** [review]

**Comment by amanmahajan7** [comment]
I think sorting and infinite scroll features are not compatible and users must come up with a strategy if they want to add both. This example mainly demonstrates how to implement infinite scrolling using the scroll event

**Pull request merged**